lib/utils/combineJSON.js

```diff
@@ -61,14 +61,14 @@ function combineJSON(arr, deep, collision, source, parsers=[]) {
 
       // Iterate over custom parsers, if the file path matches the parser's
       // pattern regex, use it's parse function to generate the object
-      parsers.forEach(({pattern, parse}) => {
+      for (const {pattern, parse} of parsers) {
         if (resolvedPath.match(pattern)) {
-          file_content = parse({
+          file_content = await parse({
             contents: fs.readFileSync(resolvedPath, {encoding:'UTF-8'}),
```

Comment:
As you are converting these in async await, I think you need to use `readFile()` instead of  `readFileSync`